spec/jobs/education_form/create_spool_submissions_report_spec.rb

```diff
@@ -40,12 +40,12 @@
         data = subject.create_csv_array
         expect(data[:stem_exists]).to eq(false)
       end
+    end
 
-      it 'recognizes 1995s as STEM submission' do
-        create(:education_benefits_claim_1995s, processed_at: time.beginning_of_day)
-        data = subject.create_csv_array
-        expect(data[:stem_exists]).to eq(true)
-      end
+    it 'recognizes 1995s as STEM submission' do
+      create(:education_benefits_claim_1995s, processed_at: time.beginning_of_day)
+      data = subject.create_csv_array
+      expect(data[:stem_exists]).to eq(true)
```

Comment:
Should this be reverted?  Looks like all that has changed now is the moving of an `end`, which moves `it 'recognizes 1995s as STEM submission'` outside of the `describe '#create_csv_array'` block

Reply:
Hi @jholton that should be reverted, let me go ahead and do so.